### PR TITLE
Creating skills

### PR DESCRIPTION
**Pull request by jjhuang417**
Created the skills.jsx page and the skeleton.

### CODE REVIEW

File: pages/skills.jsx [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+const checkBoxStyling = `
+  m-2
+`;
+
+export default function skills () {
+  const [userData, setData] = useState();
+  const [userName, setName] = useState('');
+  const [projectManagerCheck, setPMCheck] = useState(false);
+  const [engineerCheck, setEngineerCheck] = useState(false);
+  const [designerCheck, setDesignerCheck] = useState(false);
```
Comment (ericjlie) [inline pages/skills.jsx:89]: There's nothing wrong with holding input value this way, but I suggest replacing these value-holding pieces of state with useRef() when possible as it reduces the number of component re-renders.

File: pages/skills.jsx [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+            {userIcon}
+            <div className={headerLink}>My Profile</div>
+          </span>
+        </div>
+      </div>
+      <div className={topHalfFormContainer}>
+        <form className={topHalfForm}>
+          <input className={shortInput} type='text' name='name' value={userName} placeholder='Name' required></input>
+          <input className={longInput} type='text' name='portfolio' value='' placeholder='Portfolio / Work Example' required></input>
+          <label className={labelStyling}></label>
```
Comment (ericjlie) [inline pages/skills.jsx]: Empty label element. Intentional?

**Review by ericjlie — APPROVED** [review]
Good work! Minor comments, the rest looks good.

File: pages/skills.jsx (hunk 1 above)
Comment (jjhuang417) [reply pages/skills.jsx:89]: Yea I need to look into useRef again and master it.

File: pages/skills.jsx (hunk 2 above)
Comment (jjhuang417) [reply pages/skills.jsx]: Taken out. Thanks for catching that.

**Pull request merged**